test/fixtures/e2e/amp-form/amp-form.ssr.html

```diff
@@ -0,0 +1,90 @@
+<!doctype html>
+<html ⚡ allow-viewer-render-template>
+<head>
+    <meta charset="utf-8">
+    <title>Forms Examples in AMP</title>
+    <link rel="canonical" href="//example.com" >
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <script async custom-element="amp-form" src="https://cdn.ampproject.org/v0/amp-form-0.1.js"></script>
+    <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.1.js"></script>
+    <style amp-custom>
+      input.user-invalid {
+          background-color: #dc4e41;
+      }
+
+      input.user-valid {
+          background-color: #4CAF50;
+      }
+
+      fieldset.user-invalid {
+          border: 3px solid #dc4e41;
+      }
+
+      fieldset.user-valid {
+          border: 3px solid #4CAF50;
+      }
+
+      form .form-message {
+          display: none;
+      }
+
+      form.user-invalid .invalid-message {
+          display: block;
+          color: #dc4e41;
+      }
+
+      form.user-valid .valid-message {
+          display: block;
+          color: #4CAF50;
+      }
+
+      form.amp-form-submitting .submitting-message {
+          display: block;
+          color: #919191;
+      }
+
+      form.amp-form-submit-success [submit-success] {
+          color: #0000ff;
+      }
+
+      form.amp-form-submit-error [submit-error] {
+          color: #dc4e41;
+      }
+  </style>
```

Comment:
Are these styles necessary? Let's make this fixture document as minimal as possible.

Reply:
done.